spec/index.md

```diff
@@ -2589,6 +2684,8 @@ These are the synchronous read messages:
         canister_id : CanisterId;
         method_name : Text;
         arg : Blob;
+        requested_certificate : Text;
```

Comment:
Should here be another type ?

CanisterSignedQuery ?

Reply:
I've renamed the `CanisterQuery` type to `CanisterSignedQuery` to not introduce a new type in the formal part of the spec.